fabric-network/test/transaction.js

```diff
@@ -301,15 +301,15 @@ describe('Transaction', () => {
 		});
 
 		it('sends proposal with long timeout', async () => {
```

Comment:
With the new behaviour, this test might be better named.
```suggestion
		it('uses endorseTimeout option as proposal timeout', async () => {
```

Reply:
Thanks for your comment. I'll reflect your advice in the source code.